qa/common/gen_ensemble_model_utils.py

```diff
@@ -31,37 +31,13 @@
 import numpy as np
 import test_util as tu
 
+from .gen_common import np_to_model_dtype
```

Comment:
We can keep this, but I wanted to make a note that I'm seeing [this type of formatting](https://github.com/triton-inference-server/client/blob/5a9bd15899d64fa199cf7801df948a5f01b76ee6/src/python/library/tritonclient/grpc/_infer_stream.py#L35) in our client repo, where the file starts with an underscore. I could not find a recommended style for Python. I do see we have other repos (Model Navigator, Model Analyzer, PyTriton) that use this naming convention. Either approach is likely fine, though consistency would be helpful.

Reply:
The `_` is mostly to distinguish the public APIs from private API. I think for these utility scripts what we have is fine since it is just an internal utility script.

Reply:
Got it. Thanks for explaining!